src/matchers/toBeAfter/index.js

```diff
@@ -1,26 +1,26 @@
-import { matcherHint, printReceived } from 'jest-matcher-utils';
-
 import predicate from './predicate';
 
-const passMessage = (received, after) => () =>
-  matcherHint('.not.toBeAfter', 'received', '') +
+const passMessage = (received, after, context) => () =>
+  context.utils.matcherHint('.not.toBeAfter', 'received', '') +
   '\n\n' +
-  `Expected date to be after ${printReceived(after)} but received:\n` +
-  `  ${printReceived(received)}`;
+  `Expected date to be after ${context.utils.printReceived(after)} but received:\n` +
+  `  ${context.utils.printReceived(received)}`;
 
-const failMessage = (received, after) => () =>
-  matcherHint('.toBeAfter', 'received', '') +
+const failMessage = (received, after, context) => () =>
```

Comment:
```suggestion
const failMessage = (received, after, { matcherHint, printReceived }) => () =>
```

etc? again, to remove diff

Reply:
It would reduce diff, but I thought it was kinda nice that the matchers all followed the same pattern of all using `context` (although will now be `utils` when I make that change).